source/dagger/gameplay/team_game/gravity.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "core/core.h"
+#include "core/system.h"
+
+using namespace dagger;
+
+namespace team_game {
+
+    struct Gravity
+    {
+        Float32 airTime = 0.f;
+        Float32 terminalVelocity = 5.f;
+    };
+
+    class GravitySystem : public System
+    {
+    private:
+        float m_GravityStrength = 0.5f;
```

Comment:
Might just be my own preference or a nitpick, but consider renaming this to m_GravitationalConstant or m_GravitationalAcceleration. Makes it easier to understand what it is at first glance :smile:

Reply:
Yeah, it behaves like an acceleration so i'll change the name accordingly

Reply:
Renamed